index.html

```diff
@@ -71,7 +98,7 @@ <h2 class="p-sub">and I&#039;m a developer
         <section id="Portfolio" class="works">
             <h1>My Recent Works</h1>
             <hr>
-            <div class="grid works-list">
+            <div id="works-list" class="grid works-list">
                 <article class="card work">
                     <div class="background">
                         <img src="imgs/Img%20Placeholder.jpg" alt="">
```

Comment:
- [ ] Kindly load the project dynamically from `projectList array` in `js`  file and delete the hard-corded ones in `index.html`. This will make it easy to add projects in future since you will be doing it in one place.